webapp-django/crashstats/tests/test_sentrylib.py

```diff
@@ -17,58 +17,46 @@
-         ('SELECT "auth_user"."is_active" FROM "auth_user" WHERE'
-          ' "auth_user"."username" = \'username\''),
-         ('SELECT "auth_user"."is_active" FROM "auth_user" WHERE'
-          ' "auth_user"."username[filtered]')),
+        'username': (
+            'SELECT "auth_user"."is_active" FROM "auth_user" WHERE'
+            ' "auth_user"."username" = \'username\''),
         # Set a user's password
-        ('password',
-         'UPDATE "auth_user" SET "password" = \'!unusable_pwd\' WHERE "auth_user"."id" = 2',
-         'UPDATE "auth_user" SET "password[filtered]'),
+        'password': (
+            'UPDATE "auth_user" SET "password" = \'!unusable_pwd\' WHERE "auth_user"."id" = 2'),
         # Update a session (includes session data, ID)
-        ('session_data',
-         ('UPDATE "django_session" SET "session_data" = \'B64Data\', WHERE'
-          ' "django_session"."session_key" = \'a_session_key\''),
-         'UPDATE "django_session" SET "session_data[filtered]'),
+        'session_data': (
+            'UPDATE "django_session" SET "session_data" = \'B64Data\', WHERE'
+            ' "django_session"."session_key" = \'a_session_key\''),
         # Store a session (includes session ID, data)
-        ('session_key',
-         ('INSERT INTO "django_session" ("session_key", "session_data") VALUES'
-          ' (\'a_session_key\', \'B64SessionData\'))'),
-         'INSERT INTO "django_session" ("session_key[filtered]'),
+        'session_key': (
+            'INSERT INTO "django_session" ("session_key", "session_data") VALUES'
+            ' (\'a_session_key\', \'B64SessionData\'))'),
         # Load data for an API token
-        ('tokens_token.key',
-         ('SELECT "tokens_token"."id", "tokens_token"."user_id",'
-          ' "tokens_token"."key", "tokens_token"."expires",'),
-         ('SELECT "tokens_token"."id", "tokens_token"."user_id",'
-          ' "tokens_token"."key"[filtered]')),
+        'tokens_token.key': (
+            'SELECT "tokens_token"."id", "tokens_token"."user_id",'
+            ' "tokens_token"."key", "tokens_token"."expires",'),
         # Update last login
-        ('auth_user.id',
-         ('UPDATE \"auth_user\" SET \"last_login\" ='
-          ' datetime.datetime(2019, 5, 3, 18, 7, 29, 852498)'
-          ' WHERE \"auth_user\".\"id\" = 123'),
-         ('UPDATE \"auth_user\" SET \"last_login\" ='
-          ' datetime.datetime(2019, 5, 3, 18, 7, 29, 852498)'
-          ' WHERE \"auth_user\".\"id\"[filtered]')),
-    )
+        'auth_user.id': (
+            'UPDATE \"auth_user\" SET \"last_login\" ='
+            ' datetime.datetime(2019, 5, 3, 18, 7, 29, 852498)'
+            ' WHERE \"auth_user\".\"id\" = 123'),
+    }
 
     @pytest.mark.parametrize(
-        'keyword,message,expected', CASES,
-        ids=tuple(case[0] for case in CASES))
-    def test_filtered_queries(self, keyword, message, expected):
+        'keyword,sql', CASES.items(),
+        ids=tuple(case for case in CASES))
```

Comment:
Does this line up right? Does `CASES.items()` return the same as `iter(CASES)`?

Reply:
In my experience, yes, and that's my understanding from the dictionary view docs: https://docs.python.org/3.6/library/stdtypes.html#dictionary-view-objects